plaso/parsers/text_plugins/winpca.py

```diff
@@ -0,0 +1,188 @@
+
+    Args:
+      structure (pyparsing.ParseResults): tokens from a parsed log line.
+
+    Returns:
+      dfdatetime.TimeElements: date and time value.
+
+    Raises:
+      ParseError: if a valid date and time value cannot be derived from
+          the time elements.
+    """
+    date_string, time_string = date_time_string.split(' ')
+    try:
+      year_string, month_string, day_of_month_string = date_string.split('-')
+      year = int(year_string, 10)
+      month = int(month_string, 10)
+      day_of_month = int(day_of_month_string, 10)
+    except (AttributeError, ValueError):
+      raise errors.ParseError('Unsupported date string: {0:s}'.format(
+          date_string))
+
+    try:
+      time_value, ms_value = time_string.split('.')
+      hours_string, minutes_string, seconds_string = time_value.split(':')
+      hours = int(hours_string, 10)
+      minutes = int(minutes_string, 10)
+      seconds = int(seconds_string, 10)
+      milliseconds = int(ms_value, 10)
+    except (AttributeError, ValueError):
+      raise errors.ParseError('Unsupported time string: {0:s}'.format(
+          time_string))
+
+    time_elements_tuple = (
+        year, month, day_of_month, hours, minutes, seconds, milliseconds)
+
+    date_time = dfdatetime_time_elements.TimeElementsInMilliseconds(
+        time_elements_tuple=time_elements_tuple)
+    date_time.is_local_time = False
+
+    return date_time
+
+  def VerifyRow(self, parser_mediator, row):
+    """Verifies if a line of the file is in the expected format.
+
+    Args:
+      parser_mediator (ParserMediator): mediates interactions between parsers
+          and other components, such as storage and dfvfs.
+      row (dict[str, str]): fields of a single row, as specified in COLUMNS.
+
+    Returns:
+      bool: True if this is the correct parser, False otherwise.
+    """
+    if len(row) < self._MINIMUM_NUMBER_OF_COLUMNS:
+      return False
+
+    try:
+      _ = self._ParseDateTime(row['datetime'])
+    except (AttributeError, ValueError):
+      return False
+
+    return '\\' in row['program']
+
+
+class WinPCADicParser(WinPCABaseParser):
+  """Parses the Windows Program Compatibility Assistant DIC files"""
+
+  NAME = 'winpca_dic'
+  DATA_FORMAT = 'Windows PCA log file'
+
+  COLUMNS = ['program', 'datetime']
+
+  _MINIMUM_NUMBER_OF_COLUMNS = 2
+
+  def ParseRow(self, parser_mediator, row_offset, row):
+    """Parses a line of the log file and produces events.
+
+    Args:
+      parser_mediator (ParserMediator): mediates interactions between parsers
+          and other components, such as storage and dfvfs.
+      row_offset (int): offset of the line from which the row was extracted.
+      row (dict[str, str]): fields of a single row, as specified in COLUMNS.
+    """
+    date_time = self._ParseDateTime(row['datetime'])
+    if date_time is None:
+      return
+
+    event_data = WinPCAEventData()
+    event_data.body = row['program']
+    event_data.last_execution_time = date_time
```

Comment:
Change to `= self._ParseDateTime(row['datetime'])` and remove block above. Change `place_holder_event:` to false in the timeliner configuration if WinPCAEventData without a last_execution_time should not yield events.